resources/views/layouts/components/body-end.blade.php

```diff
@@ -1,6 +1,6 @@
 {{-- TODO add livewire--}}
 {{--<livewire:scripts />--}}
-@if(!app()->environment('production'))
+@if(!app()->environment('production') && env('DEBUGBAR_ENABLED'))
```

Comment:
I think this was intended to minimize accidental changes to prod (see also #1519).

Note that unless the debugbar is completely collapsed, you'll never see this. So it almost seems more desirable to have when the debugbar is disabled.

Also note that the debugbar is not enabled on the stage host.

Maybe the information could be moved up into the header, where it would be offscreen for most screenshots?

![image](https://user-images.githubusercontent.com/32680403/236528630-73430448-1ad0-48f0-970b-1f09db7532a9.png)

Reply:
My original implementation actually caused this debug component to just hide on click. What do you think of that approach?

Reply:
Seems like a reasonable solution.

Reply:
Resolved in https://github.com/RetroAchievements/RAWeb/pull/1548/commits/863027ddcd630b2c14dde15145e7d2ae5b425510.